libretroshare/src/use_libretroshare.pri

```diff
@@ -28,6 +28,7 @@ equals(TARGET, retroshare):equals(TEMPLATE, lib){
 
 bitdht {
     !include("../../libbitdht/src/use_libbitdht.pri"):error("Including")
+    DEFINES *= RS_USE_BITDHT
```

Comment:
this should go in retroshare.pri, for example just before
```
rs_async_chat {
    DEFINES *= RS_ASYNC_CHAT
}
```

Reply:
done (force pushed)